src/guacenc/guacenc.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2016 Glyptodon, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef GUACENC_H
+#define GUACENC_H
+
+#include "config.h"
+
+#include <guacamole/client.h>
+
+/**
+ * The name of the codec to use by default, if no other codec is specified on
+ * the command line. This name is dictated by ffmpeg / libavcodec.
+ */
+#define GUACENC_DEFAULT_CODEC "mpeg4"
+
+/**
+ * The extension to append to the end of the input file to produce the output
+ * file name, excluding the separating period, if no other suffix is specified
+ * on the command line.
+ */
+#define GUACENC_DEFAULT_SUFFIX "mpg"
```

Comment:
Neither `GUACENC_DEFAULT_CODEC` nor `GUACENC_DEFAULT_SUFFIX` are used - they are from an earlier iteration where both of these were configurable.